src/watertap_contrib/reflo/code_demos/steady_state_flowsheets/simple_RO_unit.py

```diff
@@ -0,0 +1,4 @@
+class ROUnit:
+    def __init__(self):
+        self.product = 6000  # m3/day
+        self.power_demand = 1000  # kW
```

Comment:
WHen we replace this, we might also think about a "simplified" RO model that is intended to serve as a reduced order model for dynamic/multiperiod simulations. At the least, we can make this an IDAES unit model.

Reply:
Yeah after this PR I'll put some time into finalizing my flowsheet with the full model. Right now I'm just having an issue with the RO initialization occurring for each time step which is making the demo take a VERY long time